src/main/java/dinkplugin/notifiers/GambleNotifier.java

```diff
@@ -0,0 +1,138 @@
+import dinkplugin.message.NotificationType;
+import dinkplugin.notifiers.data.GambleNotificationData;
+import dinkplugin.notifiers.data.SerializedItemStack;
+import dinkplugin.util.ItemSearcher;
+import dinkplugin.util.ItemUtils;
+import dinkplugin.util.Utils;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.client.game.ItemManager;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.Pair;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.VisibleForTesting;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Slf4j
+@Singleton
+public class GambleNotifier extends BaseNotifier {
+    // itemName (x quantity)![ tertiaryReward!] gc: n
+    private static final Pattern GAMBLE_REGEX = Pattern.compile(
+        "^(.+?)!\\s*(.+!)?\\s+High level gamble count: (\\d+).*"
+    );
+    private static final Pattern ITEM_QUANTITY_REGEX = Pattern.compile("^(.+?)\\(x (\\d+)\\)$");
+    private static final List<String> RARE_LOOT = ImmutableList.of("dragon chainbody", "dragon med helm");
+
+    @Value
+    @VisibleForTesting
+    static class ParsedData {
+        @NonNull
+        String itemName;
+        int itemQuantity;
+        @Nullable
+        String tertiaryItem;
+        int gambleCount;
+
+        Stream<Pair<String, Integer>> itemsWithQuantities() {
+            return Stream.of(Pair.of(itemName, itemQuantity), Pair.of(tertiaryItem, 1))
+                .filter(p -> p.getLeft() != null);
+        }
+    }
+
+    @Inject
+    private ItemSearcher itemSearcher;
+
+    @Inject
+    private ItemManager itemManager;
+
+    @Override
+    public boolean isEnabled() {
+        return config.notifyGamble() && super.isEnabled();
+    }
+
+    @Override
+    protected String getWebhookUrl() {
+        return config.gambleWebhook();
+    }
+
+    public void onMesBoxNotification(String message) {
+        if (!isEnabled()) return;
+        ParsedData data = parse(message);
+        if (data != null) {
+            handleNotify(data);
+        }
+    }
+
+    @Nullable
+    @VisibleForTesting
+    static ParsedData parse(String message) {
+        Matcher matcher = GAMBLE_REGEX.matcher(message);
+        if (!matcher.matches()) return null;
+        String rawItem = matcher.group(1).trim();
+        String itemName = rawItem;
+        int itemQuantity = 1;
+        Matcher quantityMatcher = ITEM_QUANTITY_REGEX.matcher(rawItem);
+        if (quantityMatcher.matches()) {
+            itemName = quantityMatcher.group(1).trim();
+            itemQuantity = Integer.parseInt(quantityMatcher.group(2));
+        }
+        String tertiary = matcher.group(2);
+        if (tertiary != null) {
+            tertiary = tertiary.replaceAll("!", "").trim();
```

Comment:
```suggestion
            tertiary = tertiary.replace("!", "").trim();
```
avoids more expensive regex method... alternatively you can use `StringUtils.removeEnd(tertiary, "!")`